mypy/private/requirements.in

```
@@ -1,2 +1,3 @@
 click~=8.1.7
-mypy~=1.11.2
\ No newline at end of file
+mypy~=1.11.2
+colorama
```

since we test the pins are consistent with requirements.in we need to tack on a version here (not to worry, it'll get renovated)
```suggestion
colorama~=0.4.6
```